stock_shipment_management/__openerp__.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Authors: Joël Grand-Guillaume, Yannick Vaucher
+#    Copyright 2013-2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more description.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+{"name": "Shipment Management",
+ "version": "0.1",
+ "author": "Camptocamp",
+ "category": "Transportation",
```

Comment:
@lepistone This module is not there for warehouse management as we want to know when a shipment is arrived. Does a warehouse manager cares about it? IMO it's more about transportation than stock.